test/unit.fromasm.imprecise

```diff
@@ -220,10 +220,7 @@
   (i32.const 2147483647)
  )
  (func $trapping_sint_div_s (; has Stack IR ;) (result i32)
-  (i32.div_s
-   (i32.const -2147483648)
-   (i32.const -1)
-  )
+  (i32.const -2147483648)
```

Comment:
I guess it's ok form `.imprecise` files which as I understand ignore implicit traps?